oauth2-server/src/main/java/com/clouway/oauth2/token/BearerToken.java

```diff
@@ -15,21 +15,23 @@
   public final String identityId;
   public final String clientId;
   public final String email;
+  public final int  index;
   public final Set<String> scopes;
   private final DateTime expiresAt;
 
   public BearerToken() {
-    this(null, null, null, null, null, null, null);
+    this(null, null, null, null, null, null, null, 0);
   }
 
-  public BearerToken(String value, GrantType grantType, String identityId, String clientId, String email, Set<String> scopes, DateTime expiresAt) {
+  public BearerToken(String value, GrantType grantType, String identityId, String clientId, String email, Set<String> scopes, DateTime expiresAt, int index) {
```

Comment:
I see now, we call findIdentity() when return currentUser and there index will be passed from token.